rest/taskana-rest-spring/src/test/java/pro/taskana/rest/WorkbasketDefinitionControllerIntTest.java

```diff
@@ -96,46 +97,95 @@ void testExportWorkbasketFromDomain() {
 
   @Test
   void testExportWorkbasketsFromWrongDomain() {
-    ResponseEntity<List<WorkbasketDefinitionResource>> response =
-        template.exchange(
-            restHelper.toUrl(Mapping.URL_WORKBASKETDEFIITIONS) + "?domain=wrongDomain",
-            HttpMethod.GET,
-            restHelper.defaultRequest(),
-            ParameterizedTypeReference.forType(List.class));
+    ResponseEntity<List<WorkbasketDefinitionResource>> response = getExportForDomain("wrongDomain");
     assertEquals(0, response.getBody().size());
   }
 
   @Test
-  void testImportWorkbasket() throws IOException {
-    ResponseEntity<List<WorkbasketDefinitionResource>> response =
-        template.exchange(
-            restHelper.toUrl(Mapping.URL_WORKBASKETDEFIITIONS) + "?domain=DOMAIN_A",
-            HttpMethod.GET,
-            restHelper.defaultRequest(),
-            ParameterizedTypeReference.forType(List.class));
-
-    List<String> list = new ArrayList<>();
-    ObjectMapper objMapper = new ObjectMapper();
-    list.add(objMapper.writeValueAsString(response.getBody().get(0)));
-    ResponseEntity<Void> responseImport = importRequest(list);
-    assertEquals(HttpStatus.NO_CONTENT, responseImport.getStatusCode());
+  void testImportEveryWorkbasketFromDomainA() throws IOException {
```

Comment:
this test is to avoid the problems Holger had - it imports every workbasket in domain A seperately